app/helpers/patient_query_helper.rb

```diff
@@ -149,6 +150,9 @@ def patients_by_linelist(current_user, workflow, tab, jurisdiction)
 
       current_user.patients&.where(isolation: true, purged: false)
     else
+      return current_user.patients&.monitoring_active(true) if tab == :active
```

Comment:
```suggestion
      return current_user.patients&.monitoring_open if tab == :active
```

the `monitoring_acttive` scope is mainly used for analytics (just realized that that scope should include a `where(purged: false)` as well)

Reply:
Good call. https://github.com/SaraAlert/SaraAlert/pull/937/commits/187295c0d8009a6fd6eb5a2f21dbcd9e62052439

As for the issue with `monitoring_active`, would you mind opening up a separate PR with that change? That seems like it would totally change a lot of the analytics numbers.

Reply:
hopefully the numbers shouldn't really change because patients should never really be in a state where monitoring is true and purged is true, but here is the PR: https://github.com/SaraAlert/SaraAlert/pull/944